docusaurus/docs/dev-docs/preview.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Setting up the Preview feature
+description: Learn to set up the Preview feature to link your front end application to Strapi's Content Manager Preview feature.
+displayedSidebar: devDocsSidebar
+tags:
+- content manager
+- preview
+- configuration
+---
+
+# Setting up the Preview feature <BetaBadge />
+
+Strapi's Preview feature enables previewing content in a frontend application directly from the Strapi admin panel. 
+
+The present page describes how to set up the Preview feature in Strapi. Once set up, the feature can be used as described in the [User Guide](/user-docs/content-manager/previewing-content).
+
+:::prerequisites
+* The following environment variables must be defined in your `.env` file, replacing example values with appropriate values:
+
+  ```bash
+  CLIENT_URL=https://your-frontend-app.com
+  PREVIEW_SECRET=your-secret-key # optional, required with Next.js draft mode
+  ```
+
+* A front-end application for your Strapi project should be already created and set up.
+:::
+
+## Configuration components
+
+The Preview feature configuration is stored in the `preview` object of [the `config/admin` file](/dev-docs/configurations/admin-panel) and consists of 3 key components:
+
+### Activation flag
+
+Enables or disables the preview feature:
+```javascript title="config/admin.ts|js" {3}
+// …
+preview: {
+  enabled: true,
+    // …
+}
+// …
+```
+
+### Allowed origins
+
+Controls which domains can access previews:
```

Comment:
I'm a little bit confused by this wording. What this setting does is allow these domains to be loaded in iframes in your admin (it updates the CSP policy's `frame-src` directive).

Reply:
looks like "are allowed" is missing :)